.circleci/config.yml

```diff
@@ -7,9 +7,13 @@
 #     + DOCKERHUB_PASSWORD
 #     + AWS_ACCESS_KEY_ID (for S3)
 #     + AWS_SECRET_ACCESS_KEY (for S3)
+#     + TELEPRESENCE_KUBECEPTION_TOKEN
```

Comment:
Variables that Telepresence isn't using at runtime should probably have `DEV_` in them.